package.json

```diff
@@ -29,7 +29,7 @@
     "test:ci": "nyc --silent lerna run --parallel test:ci"
   },
   "engines": {
-    "node": ">=16.18"
+    "node": ">=20.5"
```

Comment:
Updated to match package.json in the individual workspaces. I don't think this one matter much because the ci builds reference the files in `.github`.